### PR TITLE
[Troubleshooting] Failed background workers

### LINKED ISSUE #1374: Document fix for `failed to start a background worker`
Troubleshooting for the error message `failed to start a background worker` is one of the most-viewed pages on our forum. Add this information to the docs.

### PR DESCRIPTION
**Pull request by charislam**
# Description

How to fix 'failed background workers' error

# Links

Fixes #1374 

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Are procedure and highlight tags used appropriately?
- [ ] Has the index been updated appropriately?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] Are all links provided in reference style, and resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: _troubleshooting/timescaledb/background-worker-failed-start.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<!---
+* Keep this section in alphabetical order
+* Use this format for writing troubleshooting sections:
+ - Cause: What causes the problem?
+ - Consequence: What does the user see when they hit this problem?
+ - Fix/Workaround: What can the user do to fix or work around the problem? Provide a "Resolving" Procedure if required.
+ - Result: When the user applies the fix, what is the result when the same action is applied?
+* Copy this comment at the top of every troubleshooting page
+-->
+
```
Comment (Loquacity) [inline _troubleshooting/timescaledb/background-worker-failed-start.md:30]: Missing title?

File: _troubleshooting/timescaledb/background-worker-failed-start.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+You might see this error message in the logs if background workers aren't
+properly configured.
+
```
Comment (Loquacity) [inline _troubleshooting/timescaledb/background-worker-failed-start.md:33]: Missing error message?

File: _troubleshooting/timescaledb/background-worker-failed-start.md (hunk 1 above)
Comment (charislam) [reply _troubleshooting/timescaledb/background-worker-failed-start.md:30]: This is for the reorganized troubleshooting section! (I created a bit of a maintenance headache for myself by being too quick to reorganize the troubleshooting entries. After having to update them a few times to keep them in sync, I decided the easiest way was to merge the thing and make a parallel edit every time we have a PR that edits the current troubleshooting pages.)

In the proposed new troubleshooting format, you don't need the title and error message in the body! They'll be pulled from frontmatter.

The second file is the page that will be changed on our current live site and it has both in the message body :)

Comment (Loquacity) [reply _troubleshooting/timescaledb/background-worker-failed-start.md:30]: Right! OK 👍

**Review by Loquacity — APPROVED** [review]

**Pull request merged**